### ISSUE TITLE
Idea : Replay a Drive

### ISSUE THREAD

**Issue by skreu** [CarlosCuezva/dashboards-Grafana-Teslamate#34]
Hi, 

I would love to be able to stream (or replay-stream?) a drive. Today I can see with beautiful graphs the global trip, but I would like to be able to see live (or re-live) the position with a map, a power gauge, battery level and more but in real time.

Don't specifically need to have a curve, just be able to watch a replay-stream of power, speed, elevation, position and more. Do you think this is possible using Teslamate and Grafana ? I know the database positions gives all that information with time but didn't find a solution yet.

Thanks very much :)

**Comment by CarlosCuezva** [comment]
Yes, it will be possible like the "Current Charge View" dashboard, but with graphs and trip data.
